packages/vega-scenegraph/src/util/aria.js

```diff
@@ -6,6 +6,7 @@ const ARIA_LABEL = 'aria-label';
 const ARIA_ROLE = 'role';
 const ARIA_ROLEDESCRIPTION = 'aria-roledescription';
 const DEFAULT_ROLE = 'graphics-symbol';
```

Comment:
Should we rename this to `MARK_ROLE` or `INDIVIDUAL_MARK_ROLE`?

Reply:
Or maybe rename to `SYMBOL_ROLE`? Also which of these should be used for guides?

Reply:
Guides should use some role that does not have children so that VoiceOver does not read `empty X` after reading the description.